.github/workflows/CI.yml

```
@@ -292,15 +294,16 @@ jobs:
 
   test-mongodb:
     runs-on: ${{ matrix.os }}
-    continue-on-error: ${{ matrix.julia-version == '1.12-nightly' || matrix.julia-version == 'nightly' }}
+    continue-on-error: ${{ matrix.julia-version == '1.13-nightly' || matrix.julia-version == 'nightly' }}
```

```suggestion
    continue-on-error: ${{ contains(matrix.julia-version, 'nightly') }}
```
That would be easier to maintain.